kuma/wiki/kumascript.py

```diff
@@ -90,43 +95,28 @@ def get(document, base_url, cache_control=None, timeout=None):
         site = Site.objects.get_current()
         base_url = 'http://%s' % site.domain
 
-    body, errors = None, None
-
-    try:
-        # Assemble some KumaScript env vars
-        # TODO: See dekiscript vars for future inspiration
-        # http://developer.mindtouch.com/en/docs/DekiScript/Reference/
-        #   Wiki_Functions_and_Variables
-        path = document.get_absolute_url()
-
-        # TODO: Someday merge with _get_document_for_json in views.py
-        # where most of this is duplicated code.
-        env_vars = dict(
-            path=path,
-            url=urljoin(base_url, path),
-            id=document.pk,
-            revision_id=document.current_revision.pk,
-            locale=document.locale,
-            title=document.title,
-            slug=document.slug,
-            tags=list(document.tags.names()),
-            review_tags=list(document.current_revision.review_tags.names()),
-            modified=time.mktime(document.modified.timetuple()),
-        )
-
-        body, errors = _post(document.html, env_vars, cache_control, timeout)
-
-    except Exception as exc:
-        # Last resort: Something went really haywire. Kumascript server died
-        # mid-request, or something. Try to report at least some hint.
-        errors = [
-            {
-                "level": "error",
-                "message": "Kumascript service failed unexpectedly: %s" % exc,
-                "args": ["UnknownError"],
-            },
-        ]
-    return (body, errors)
+    # Assemble some KumaScript env vars
+    # TODO: See dekiscript vars for future inspiration
+    # http://developer.mindtouch.com/en/docs/DekiScript/Reference/
+    #   Wiki_Functions_and_Variables
+    path = document.get_absolute_url()
+
+    # TODO: Someday merge with _get_document_for_json in views.py
```

Comment:
Remove this TODO as well. ``_get_document_for_json`` is gone, maybe is ``build_json_data`` now, but in any case, it seems like it should not be a priority for future devs with lots of time.